contrib/mesos/pkg/executor/service/service.go

```diff
@@ -173,6 +173,19 @@ func (s *KubeletExecutorServer) runKubelet(
 		kcfg.DockerDaemonContainer = "" // don't move the docker daemon into a cgroup
 		kcfg.Hostname = kcfg.HostnameOverride
 		kcfg.KubeClient = apiclient
+
+		// taken from KubeletServer#Run(*KubeletConfig)
+		clientConfig, err := s.CreateAPIServerClientConfig()
+		if err == nil {
+			kcfg.KubeClient, err = client.New(clientConfig)
```

Comment:
if the intent is to create a separate client for events, then why overwrite `kcfg.KubeClient` here?